src/api/admin.js

```diff
@@ -0,0 +1,91 @@
+const data = {}
+const writeFile = ()=>{
+  const fileData = JSON.stringify(data)
+  fs.writeFile('./data.db', fileData,  (err)=>{
+    if(err) {
+      return console.log('error', err)
+    }
+    console.log('file write success')
+  })
+}
+
+fs.readFile('./data.db', (err, file)=>{
+  if(err){
+    return console.log('error', err)
+  }
+  const fileData = JSON.parse(file || '{}')
+  data.serviceKeys = fileData.serviceKeys || []
+})
+
+app.post('/api/admin/serviceHostsKeys', (req, res)=>{
```

Comment:
app is undefined here. Did you test your code? I don't see how this would work.
```
const app = express.Router()
app.post('/serviceHostKeys'....
```

Please look at comments below also.